datasets/msra_ner/README.md

```diff
@@ -19,6 +19,55 @@ task_ids:
 - named-entity-recognition
 paperswithcode_id: null
 pretty_name: MSRA NER
+train-eval-index:
+- config: msra_ner
+  task: token-classification
+  task_id: entity_extraction
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    tokens: tokens
+    ner_tags: tags
+  metrics:
```

Comment:
Same comment as in ConLL about using `seqeval` metric as the default. These ones won't work for NER tasks, because `evaluate` / `datasets` use `sklearn` for e.g. `f1` and that needs to be extended for sequence labelling tasks